extensions/wikia/Answers/AttributionCache.class.php

```diff
@@ -359,50 +390,51 @@ public static function purgeArticleContribs(
 		WikiPage $article, User $user, $text, $summary, $minoredit, $watchthis, $sectionanchor,
 		$flags, $revision, Status &$status, $baseRevId
 	): bool {
-		if(count($status->errors) == 0) {
-			AttributionCache::getInstance()->purge($article->getTitle(), $user);
+		if ( count( $status->errors ) == 0 ) {
+			AttributionCache::getInstance()->purge( $article->getTitle(), $user );
 		}
+
 		return true;
 	}
 
 	/**
 	 * hook: TitleMoveComplete
 	 */
-	public static function purgeArticleContribsAfterMove($oldTitle, $newTitle, $user) {
-		AttributionCache::getInstance()->purge($oldTitle, $user);
+	public static function purgeArticleContribsAfterMove( $oldTitle, $newTitle, $user ) {
+		AttributionCache::getInstance()->purge( $oldTitle, $user );
+
 		//AttributionCache::getInstance()->purge($newTitle, $user);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

[project-id]: wikia:app